pyproject.toml

```
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "granite-suite"
+version = "0.1.14"
+description = "granite is a collection of software to call, filter and work with genomic variants."
+authors = ["Michele Berselli <berselli.michele@gmail.com>", "Phil Grayson"]
+readme = "README.md"
+license = "MIT"
+repository = "https://github.com/dbmi-bgm/granite"
+homepage = "https://github.com/dbmi-bgm/granite"
+classifiers = [
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: POSIX :: Linux',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3'
+]
+packages = [
+    { include="granite" }
+]
+
+
+[tool.poetry.dependencies]
+python = "^3.6"
+pysam = ">=0.15.0"
+bitarray = ">=1.2.0"
+pytabix = ">=0.0.2"
+h5py = ">=2.10.0"
+numpy = ">=1.18.0"
+matplotlib = "==3.3.4"
+pytest = "*"
```

I am guessing pytest is only a dev dependency (needed only at testing time?). The following is untested, but is probably what you want:
```suggestion

[tool.poetry.dev-dependencies]
pytest = "*"
```
If you take the suggestion, you'll want to do `poetry update pytest` to get it to change the lock file in a minimally appropriate way. (You could also just do `poetry lock` but sometimes for an incremental change `poetry update <libraries>` is better. You can also not specify the library and it will figure it out, but sometimes it changes other things you don't want to re-test.)